lib/js/consts/vuetifyTheme.const.ts

```diff
@@ -0,0 +1,15 @@
+import { FUNCTIONAL_COLORS, THEME_COLORS } from './colors';
+
+export const VUETIFY_THEME = {
+	themes: {
+		light: {
+			primary: THEME_COLORS.PRIMARY,
+			secondary: THEME_COLORS.SECONDARY,
+			error: FUNCTIONAL_COLORS.DANGER,
+			success: FUNCTIONAL_COLORS.SUCCESS,
+			warning: FUNCTIONAL_COLORS.WARNING,
+			anchor: 'none',
```

Comment:
> Invalid property value

Reply:
that was the point, we don't want to set color to all links in an application